whois-nrtm4/src/main/java/net/ripe/db/nrtm4/SnapshotFileGenerator.java

```diff
@@ -45,51 +48,56 @@ public SnapshotFileGenerator(
         this.nrtmFileStore = nrtmFileStore;
     }
 
-    public Optional<PublishableSnapshotFile> createSnapshot(final NrtmSource source) throws IOException {
+    public List<PublishableSnapshotFile> createSnapshots() throws IOException {
         // Get last version from database.
-        final Optional<NrtmVersionInfo> lastVersion = nrtmVersionInfoRepository.findLastVersion(source);
-        NrtmVersionInfo version;
+        final Optional<NrtmVersionInfo> lastVersion = nrtmVersionInfoRepository.findLastVersion();
+        final List<NrtmVersionInfo> versions = new ArrayList<>();
```

Comment:
Could we change the name of these variable, to specify that we are storing a version per source? maybe sourcesVersion instead of version? or something like this?

Reply:
done